egov-indexer/notification-order-indexer.yml

```
@@ -0,0 +1,43 @@
+      configKey: INDEX
+      indexes:
+        - name: order-notification-view
+          type: general
+          id: $.id
+          isBulk: false
+          jsonPath: $.orderAndNotification
+          customJsonMapping:
+            indexMapping:
+              {
+                "Data":
+                  { "orderNotification": { } },
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)